doc/source/serve/production.md

```diff
@@ -460,9 +460,18 @@ deployment_statuses:
 
 ## Updating Your Serve Application in Production
 
-You can also update your Serve applications once they're in production. You can update the settings in your config file and redeploy it using the `serve deploy` command.
+You can also update your Serve applications once they're in production. You can update the settings in your config file and redeploy it using the `serve deploy` command. You can also add new deployment settings or remove old deployment settings from the config. This is because `serve deploy` is **idempotent**. Your Serve application's config always matches the latest config you deployed successfully– regardless of what config files you deployed before that.
 
-Let's use the `FruitStand` deployment graph [from an earlier section](fruit-config-yaml) as an example. All the individual fruit deployments contain a `reconfigure()` method. [This method allows us to issue lightweight updates](managing-deployments-user-configuration) to our deployments by updating the `user_config`. These updates don't need to tear down the running deployments, meaning there's less downtime as the deployments update.
+### Lightweight Config Updates
+
+Lightweight config updates modify running deployment replicas without tearing them down and restarting them, so there's less downtime as the deployments update. Changing `num_replicas`, `autoscaling_config`, and/or `user_config` in an entry of the `deployments` section of the config file is considered a lightweight config update, and won't tear down the replicas for that deployment.
+
```

Comment:
Lightweight config updates modify running deployment replicas without tearing them down or restarting them, so there's less downtime on deployments update. A config file entry under `deployments`, such as `num_replicas`, `autoscaling_config`, or `user_config`, is considered a lightweight config update, therefore it won't tear down the replicas. Nor does it require a redeployment.